File: Asocioacion/php/primerEjercicio/MediosPago.php
<?php

class MediosPago{

    private string $nombreMedio;

    public function __construct(string $nombreMedio){
        
        $this->nombreMedio = $nombreMedio;

    }

    //Nombre del medio de pago
    public function getNombreMedio(){
        return $this->nombreMedio;
    }
    public function setNombreMedio($nombreMedio){
        $this->nombreMedio = $nombreMedio;
    }

}

?>

File: Asocioacion/python/primerEjercicio/MediosPago.py
class MediosPago:
    
    c = 0
    
    def __init__(self,nombreMedio):
        self.nombreMedio = nombreMedio
        self.__class__.c=1
    
    #Nombre del medio de pago
    
    def getnombreMedio(self):
        return self.nombreMedio
    
    def setNombreMedio(self,nombreMedio):        
        self.nombreMedio = nombreMedio